src/test/scala/sorm/test/general/EntityWithNoPrimitivesSuite.scala

```diff
@@ -24,7 +24,7 @@ class EntityWithNoPrimitivesSuite extends FunSuite with ShouldMatchers {
       .whereEqual("id", 1)
       .fetchOne()
       .map(a => a.copy(genres = Set("rock","rnb"), a = Set("myfoo"), b = Set("mybar")))
-      .map(db.save)
+      .map(a => db.save(a, false))
```

Comment:
@danicuki 
This indicates a broken backwards compatibility. Better introduce the feature as a `saveDelayed` method on `Instance` instead of the optional `delayed` parameter to the `save` method as it's done currently.